boards/st/stm32h7s78_dk/stm32h7s78_dk.dts

```diff
@@ -173,6 +175,64 @@
 	pinctrl-names = "default";
 };
 
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* Set the partitions with first MB to make use of the whole Bank1 */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(64)>;
+		};
+	};
+};
+
+&xspi2 {
+	pinctrl-0 = <&xspim_p2_clk_pn6 &xspim_p2_ncs1_pn1
+		     &xspim_p2_io0_pn2 &xspim_p2_io1_pn3
+		     &xspim_p2_io2_pn4 &xspim_p2_io3_pn5
+		     &xspim_p2_io4_pn8 &xspim_p2_io5_pn9
+		     &xspim_p2_io6_pn10 &xspim_p2_io7_pn11
+		     &xspim_p2_dqs0_pn0>;
+	pinctrl-names = "default";
+
+	status = "okay";
+
+	mx25uw25645: xspi-nor-flash@70000000 {
+		compatible = "st,stm32-xspi-nor";
+		reg = <0x70000000 DT_SIZE_M(128)>; /* 1 Gbits */
+		ospi-max-frequency = <DT_FREQ_M(50)>;
+		spi-bus-width = <XSPI_OCTO_MODE>;
+		data-rate = <XSPI_DTR_TRANSFER>;
+		status = "okay";
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			slot0_partition: partition@0 {
+				label = "image-0";
+				reg = <0x00000000 DT_SIZE_K(416)>;
+			};
+			slot1_partition: partition@80000 {
+				label = "image-1";
+				reg = <0x00080000 DT_SIZE_K(416)>;
+			};
+			scratch_partition: partition@e0000 {
+				label = "image-scratch";
+				reg = <0x000e0000 DT_SIZE_K(64)>;
+			};
+			storage_partition: partition@f0000 {
+				label = "storage";
+				reg = <0x000f0000 DT_SIZE_K(64)>;
+			};
```

Comment:
Wrong offsets (0x00080000 is not 416k)

Reply:
Yes, using the same partitioning as the nucleo_h7s3l8